src/media-segment-request.js

```diff
@@ -168,11 +168,14 @@ const handleInitSegmentResponse =
 
   segment.map.bytes = new Uint8Array(request.response);
 
-  if (isLikelyWebmData(segment.map.bytes)) {
+  const type = detectContainerForBytes(segment.map.bytes);
+
+  if (type !== 'mp4') {
```

Comment:
A transport stream can still have a media initialization section, thus, an EXT-X-MAP tag: https://tools.ietf.org/html/draft-pantos-http-live-streaming-23#section-3.2

Reply:
Won't that mean that the map is mp4 though?

Reply:
The map would be a transport stream too, just with PAT and PMT

Reply:
The problem with that though is that we won't be able to parse it as we assume that the map is mp4 below. I will add a TODO comment specifying that we should also handle ts segment maps.